python/cudf/cudf/tests/test_index.py

```diff
@@ -141,6 +141,7 @@ def test_categorical_index():
 def test_pandas_as_index():
     # Define Pandas Indexes
     pdf_int_index = pd.Int64Index([1, 2, 3, 4, 5])
+    pdf_uint_index = pd.UInt64Index([1, 2, 3, 4, 5], dtype="uint64")
```

Comment:
Hoe come `UInt64Index` needs a dtype param? `uint64` is in the name of the class... ? Maybe we could default `UInt64Index` with a dtype of `uint64`?